tests/functional/services/policy_engine/conftest.py

```diff
@@ -42,14 +42,14 @@ def read_expected_content(module_path, filename):
 
 
 @pytest.fixture
-def expected_content(request) -> Callable[[str], Dict]:
+def expected_content(request) -> Callable:
     """
     Returns method that will load expected vulnerability response json for a given image_digest
     :rtype: Callable[[str], Dict]
     :return: method that loads expected response json
     """
 
-    def get_expected_content(filename) -> Dict:
```

Comment:
Should probably change this to Union[List, Dict]

Reply:
Yep, I think it might be okay to leave it open since the type enforcement is merely a hint.